packages/account/src/Sections/Security/Passwords/__tests__/deriv-password.spec.tsx

```diff
@@ -1,27 +1,13 @@
 import React from 'react';
-import { render, screen, waitFor, fireEvent, queryByText } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import DerivPassword from '../deriv-password';
```

Comment:
need to change the order of imports here

Reply:
agree